packages/cli/internal/pkg/cli/format/text_tabular.go

```diff
@@ -21,7 +21,8 @@ const (
 func (f *TextTabular) Write(o interface{}) {
```

Comment:
Could you please rename it to just "tabular"? It's a bit confusing whether it text or tabular. Especially taking into account that we already have tabular.

Reply:
Or even better to just "table"?

Reply:
Adjusted